pkg/cgroups/cgroupid.go

```diff
@@ -10,12 +10,14 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// CgroupID is a cgroup id to path map/cache.
```

Comment:
I don't fully understand this either 😉

Reply:
Hmm... fair enough. Let's try to clarify it then.